source/logging/logging.rst

```diff
@@ -149,7 +149,7 @@ driver-specific messages they produce.
      - Environment Variable
 
    * - command
-     - `Command Monitoring <../command-monitoring/command-monitoring.rst>`__
+     - `Command Monitoring <../command-logging-and-monitoring/command-logging-and-monitoring.rst>`__
```

Comment:
@kmahar: Was renaming the Command Monitoring spec (and directory) necessary? I imagine a Logging section is going to be added to multiple specs down the line as those DRIVERS tickets get implemented, so I'm curious if there was anything special about Command Monitoring that necessitated this. FWIW, the DRIVERS component is still called [APM](https://jira.mongodb.org/issues/?jql=project%20%3D%20DRIVERS%20AND%20component%20%3D%20APM).

@prestonvasquez: Note that there are still many more instances of `command-monitoring/` throughout the repository. If we're going to keep the spec path as-is, I think it'd be worth fixing those in this PR as well.

This is a nit, but one other side effect is folks may need to update any scripts they have to sync spec tests. The directory rename seems like something worth mentioning in downstream changes for [DRIVERS-1677](https://jira.mongodb.org/browse/DRIVERS-1677) (assuming that's the relevant ticket for https://github.com/mongodb/specifications/commit/d4f83d83a380a1863094c3f7bcb5d27b5963a2c9).

Reply:
There was no technical reason requiring the name change, but I did it because the spec was solely about emitting command diagnostics to begin with, and so it seemed reasonable to add logging to the title to clarify that the spec now covers both types of diagnostic events, monitoring events and log messages. 
(as an aside, I never thought it made much since to call that component APM as the only reference to "APM" at all in the spec is a sentence that mentions the events could integrate with APM libraries. I'd advocate for renaming the component CLAM now.) 
if the messages were part of some spec with a more general title e.g. "Commands" I would not have changed the title. and along those lines, for other specs, I wasn't planning to add logging to the title, since they already have titles that encompass their subject areas as a whole, e.g. I think it follows that the server selection spec would contain server selection log messages.

CMAP does have "monitoring" in the title I suppose which could be an argument to adding logging to the title there; but both since that is such a well-established acronym within DBX and also since the title already references "connection pooling" broadly I think that one should be left behind. 

DRIVERS-1673 is actually the ticket to cover command logging specifically. I updated the downstream changes section there to mention the rename occurred.

Reply:
assuming we're keeping the name change, can we update the title in this row to also be "command logging and monitoring" to match the spec?

Reply:
![](https://media.tenor.com/XEjoT8-7efoAAAAC/rilakumma-kawaii.gif)

SGTM. Since the title/path will remain as-is, can one of you take care of renaming all of the other `command-monitoring/` links within the repository?

Reply:
@jmikola I will start working on that tomorrow.